discord/opus.py

```diff
@@ -19,16 +45,18 @@
     EncoderStruct,
     EncoderStructPtr,
     InvalidArgument,
-    OK,
     OpusError,
     OpusNotLoaded,
     SignalCtl,
-    _OpusStruct,
     _err_lt,
     _err_ne,
     _lib,
     _load_default,
+    _OpusStruct,
     band_ctl,
+    c_float_ptr,
```

Comment:
just so this won't be forgotten: these three aren't imported when `shim_maker.py` is run on Windows